process/graduation_checklist.md

```diff
@@ -0,0 +1,55 @@
+# ASWF Project Graduation Template
+
+Text below in _italics_ are guidelines for answering the various
+questions.
+
+- **Project Name**:
+
+- **Criteria**
+
+  - **Demonstrate having a healthy number of committers from a diverse
+    contributor base.**
+
+    _Showcase the total number of committers as well as the diversity
+    of committers from an organizational standpoint. Link to Pull
+    Request reviews/approvals, or other indications of committer-level
+    activities. Each ASWF project is on [LF Dev Analytics] which is a 
+    good resource to use for this._
+ 
+  - **Demonstrate a substantial ongoing flow of commits and merged
+    contributions, authored by a healthy number of diverse
+    contributors.**
+  
+    _Showcase the total number of contributors since the project
+    entered incubation, as well as the diversity of contributors from
+    an organizational standpoint. Pointing to GitHub or Dev Analytics
+    graphs are good ideas ( i.e.
+    https://github.com/imageworks/OpenColorIO/graphs/contributors ).
+    Feel free to add other context if relevant. 
+    Each ASWF project is on [LF Dev Analytics] which is a 
+    good resource to use for this._
+
+  - **Document current project owners and current and emeritus
+    committers in a COMMITTER file or similarly visible system. A copy
+    of the project’s charter (or other authorized governance document)
+    will be included or linked to in visible location.**
+  
+    _Link to GOVERNAME.md, COMMITTERS.md, and TSC Charter_
+
+  - **Have achieved and maintained a Core Infrastructure Initiative
+    Best Practices Badge.**
+
+    _Link to Best Practices Badge - feel free to add other context if
+    relevant_
+
+  - **Have a technical lead appointed for representation of the project
+    to the TAC.**
+  
+    _Indicate who is the TSC Chairperson_
+
+  - **Have a completed and presented to the TAC an initial license scan
+    of the project’s codebase.**
+  
+    _Provide a copy of the latest license scan to the TAC_
```

Comment:
I don't know how to run a license scan for OpenCue or find our latest license scan, I just know that one was run prior to us transferring the repo. Does this answer need more details on how to find that or who to contact?

Reply:
@swinslow Does these on a regular basis for all of the ASWF projects. He did do one initially before the code came into the ASWF repo which spot on clean IIRC :-)

Reply:
Is there a preferred email address or something we should use here for requesting a copy of that report? Doc says "provide a copy" but it's not clear how to obtain that.

(This is more about the doc and future users here than anything OpenCue-specific.)